src/main/resources/logback.xml

```diff
@@ -63,4 +63,7 @@
   <!-- Suppress an inconsequential IOException that is logged at debug level. -->
   <logger name="org.glassfish.grizzly" level="info" />
 
+  <!-- optional logging configuration to define your own custom logging needs -->
+  <include optional="true" resource="logback-addendum.xml" />
```

Comment:
1. Would it be easier to use a file here and not a resource? That would enable using this with the "one-jar" as the only thing on the class path and the log file outside. Also, as a developer you normally start OTP where your data files reside, then you can maintain a custom log config there as well?

2. Should we go for a easier name to remember - or is the "logback-addendum.xml" the defacto standard? What about just "otp-logback.xml" ?

Reply:
Thanks for comments! Some thoughts:
- For us, using a resource seems very convenient, as we wish to develop the logging extension under version control.
- logback has another standard log called logback-test.xml . That's why I suggested similarly built and quite self-documenting file name with addendum postfix

Reply:
Are you doing this in your downstream fork of OTP, it that why you want this extension? Maybe, we can do both - I would find it useful it it was on the application path, not on the classpath at least. We can discuss this at todays meeting.

Reply:
Here is the doc for the logback file include: https://logback.qos.ch/manual/configuration.html#fileInclusion 

You can decide if you need both, but at least the file version is very useful: 
```
  <!-- optional logging configuration to define your own custom logging needs-->
  <include optional="true" file="logback-include-extensions.xml" />
  <include optional="true" resource="logback-include-extensions.xml" />  
```
I think we can move som of the existing config into an example include file later.

Reply:
I ended up to the same conclusion: we should have the standard log configuration, organization specific version controlled extension and finally a developer/branch/whatever specific local configuration which can be easily modified for temporary needs.

I will modify the pull request to include both options.

Reply:
I tested file extension and it is indeed most convenient. We can maintain it it the version control of the databuild environment repo, and developers can add their custom extensions to local runtime folder.